src/test/java/de/medizininformatikinitiative/torch/rest/FhirControllerTest.java

```diff
@@ -180,4 +185,21 @@ void blankRequestBodyTriggersBadRequest() {
                     .jsonPath("$.resourceType").isEqualTo("OperationOutcome");
         }
     }
+
+    @Nested
+    class Validator {
+        @Test
+        void invalidCrtdlTriggersBadRequest() throws ValidationException {
+            ExtractDataParameters params = new ExtractDataParameters(CrtdlFactory.empty(), Collections.emptyList());
+            when(extractDataParametersParser.parseParameters(any())).thenReturn(params);
+            when(validator.validate(any())).thenThrow(new ValidationException("Invalid CRTDL"));
+
+            var response = client.post().uri("/fhir/$extract-data").contentType(MediaType.APPLICATION_JSON).bodyValue("{}").exchange();
+
+            response.expectStatus().isBadRequest().expectHeader().contentType("application/fhir+json")
+                    .expectBody()
+                    .jsonPath("$.resourceType").isEqualTo("OperationOutcome")
+                    .jsonPath("$.issue[0].diagnostics").isEqualTo("IllegalArgumentException: Invalid CRTDL");
```

Comment:
The `diagnostics` should not contain any Java related artefacts like class names. So I would just remove `IllegalArgumentException`. "Invalid CRDTL" is good enough.